README.md

```diff
@@ -19,6 +19,10 @@ to open cypress UI console
 to run in a headless mode
 `CYPRESS_admin_password='value' npx cypress run`
 where value is the password ( can be found in digital-platform slack channel topic )
+To run using the single sign-on (SSO)
+`ocpl_username='ocpl-test'`
+`ocpl_password='$@P13nt!'`
```

Comment:
oops, please remove exposed password = )

also, just update the existing instruction since for now you need to provide both 
`CYPRESS_admin_password='value' CYPRESS_ocpl_password='value' npx cypress run`